docker-compose.yml

```diff
@@ -24,7 +23,7 @@ services:
       - 4040
       - 5000 # ${WEBAPP_PORT}
     volumes:
-      - /AiogramShopBot:/bot/data
+      - ./AiogramShopBot:/bot/data
```

Comment:
Do not add any changes to the docker compose file.

Reply:
reverted